targets/ESP32/_common/ESP32_C61_DeviceMapping.cpp

```
@@ -0,0 +1,81 @@
+int8_t Esp32_SPI_DevicePinMap[MAX_SPI_DEVICES][Esp32SpiPin_Max] = {
+    // SPI1 - use defaults
+    {MSPI_IOMUX_PIN_NUM_MOSI, MSPI_IOMUX_PIN_NUM_MISO, MSPI_IOMUX_PIN_NUM_CLK},
+    // SPI2 - no pins assigned
+    {-1, -1, -1}};
+
+//  Serial
+//  2 devices COM1,COM2 ( UART_NUM_0, UART_NUM_1 )
+//  Map pins  Tx, RX, RTS, CTS
+//  Set pins to default for UART_NUM_0
+// others assign as NONE because the default pins can be shared with serial flash and PSRAM
+int8_t Esp32_SERIAL_DevicePinMap[UART_NUM_MAX][Esp32SerialPin_Max] = {
```

Clang (14.0.6)

[warning] variable 'Esp32_SERIAL_DevicePinMap' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)